backend/app/requests/adequacy.py

```diff
@@ -58,18 +58,23 @@ def adequacy_request(app, flask_request, engine):
     except json.JSONDecodeError:
         raise InvalidFormat(message='Invalid JSON format.')
 
-    if 'provider_ids' not in request:
-        raise InvalidFormat(message='Invalid format. Could not find provider_ids.')
+    if 'provider_ids' not in request and 'providers' not in request:
+        raise InvalidFormat(message='Invalid format. Could not find provider information.')
     if 'service_area_ids' not in request:
         raise InvalidFormat(message='Invalid format. Could not find service_area_ids.')
 
-    provider_ids = request['provider_ids']
+    if 'providers' in request:
+        addresses = request['providers']
```

Comment:
what's the difference between a provider and a provider id? why does one require fetch_address_from_ids and one is just addresses?

Reply:
are they addresses or lat longs?